mkdocs.yml

```diff
@@ -30,38 +30,39 @@ theme:
 nav:
   - index.md
   - Understanding eds-scikit:
-    - project_description.md
-    - A gentle demo: functionalities/generic/introduction.ipynb
+      - project_description.md
+      - A gentle demo: functionalities/generic/introduction.ipynb
   - Functionalities:
-    - Connectors: functionalities/generic/io.ipynb
-    - Phenotyping:
-      - functionalities/phenotyping/working-with-codes.md
-      - functionalities/phenotyping/diabetes.md
-      - functionalities/phenotyping/suicide_attempts.md
-    - Patient's Course:
-      - ICU: functionalities/patients-course/is_icu.md
-      - Emergency Units: functionalities/patients-course/is_emergency.md
-      - Visit merging: functionalities/patients-course/visit_merging.md
-      - Consultation dates: functionalities/patients-course/consultation_dates.md
-    - Biology:
-      - functionalities/biology/index.md
-      - functionalities/biology/cleaning.md
-      - functionalities/biology/visualization.md
-      - functionalities/biology/vocabulary.md
-      - functionalities/biology/tutorial.ipynb
+      - Connectors: functionalities/generic/io.ipynb
```

Comment:
I'm not familiar enough with mkdocs; why were these changes necessary?

Reply:
I think it is a default pep8 formatting, it did it automatically on my VSCode when I opened/saved the file. But it does not seem to be mandatory for pre-commit, so I can just restore the older version.

Reply:
Yes, it's usually better to limit changes that are not directly related to the focus of a PR